frontend/src/plugins/i18n.ts

```diff
@@ -6,7 +6,11 @@ const localPathPrefix = "../locales/";
 
 const storage = useLocalStorage("bytebase_options", {}) as any;
 
-let locale = storage.value?.appearance?.language || navigator.language;
+const params = new URL(window.location.href).searchParams;
+let locale =
+  storage.value?.appearance?.language ||
+  params.get("locale") ||
```

Comment:
Better to narrow the value got from storage and params. But LGTM now.

Reply:
Add a `getValidLocale` function. PTAL

Reply:
And I think the params has a high priority than storage, so I reorder them.